### PR TITLE
nginx: add tests for config

### PR DESCRIPTION
**Pull request by alxndrsn**

### CODE REVIEW

File: test/test/.gitignore
Comment (alxndrsn) [inline test/test/.gitignore:1]: The files in this directory could probably move up a level so there is not `test/test`

File: test/nginx-test.dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#
+# KEEP THIS FILE AS CLOSE AS POSSIBLE TO ../nginx.dockerfile
+#
+# It is ESPECIALLY IMPORTANT to keep IDENTICAL BASE IMAGES.
+#
```
Comment (alxndrsn) [inline test/nginx-test.dockerfile:5]: One alternative to this copy/paste could be `INCLUDE+` (https://github.com/edrevo/dockerfile-plus)

Comment (lognaturel) [reply test/nginx-test.dockerfile:5]: Let's address this as part of https://github.com/getodk/central/issues/677 Then I think we can use our actual published nginx image as the base image.

**Review by lognaturel — APPROVED** [review]
I agree that having some dedicated testing of the nginx config is a great idea. I'd like to talk through whether we really need a separate dockerfile for nginx but we can revise later.

**Pull request merged**